test/test_execution.py

```diff
@@ -26,11 +17,17 @@ def test_sanity_green(temp_output_dir: str) -> None:
         capture_outputs=Outputs.ALL,
         outputs=Outputs.ALL,
         record_types=RecordTypes.ALL,
+        clobber=False,
     )
     execute(args)
-    # When runtime < sample_interval, we won't have a usage.json
-    expected_files = ["stdout", "stderr", "info.json"]
+    expected_files = [
+        SUFFIXES["stdout"],
```

Comment:
Code like this would be a good place to use the `*_SUFFIX` constants, if you keep them.